test/lib/sstable_utils.cc

```diff
@@ -152,6 +152,7 @@ compaction_manager_for_testing::wrapped_compaction_manager::wrapped_compaction_m
 
 // Must run in a seastar thread
 compaction_manager_for_testing::wrapped_compaction_manager::~wrapped_compaction_manager() {
+    tm.abort_source().request_abort();
```

Comment:
abort source doesn't like to be aborted more than once.
I suggest calling request_abort only if abort wasn't requested already.